tooling/lsp/src/requests/completion.rs

```diff
@@ -1559,7 +1559,7 @@ fn get_array_element_type(typ: Type) -> Option<Type> {
     match typ {
         Type::Array(_, typ) | Type::Slice(typ) => Some(*typ),
         Type::Alias(alias_type, generics) => {
-            let typ = alias_type.borrow().get_type(&generics);
+            let typ = alias_type.borrow().get_type(&generics).follow_bindings();
```

Comment:
Not here, I meant in the match itself we should recur in the type variable case

Reply:
Ah, I see. Let me know if I got it right this time.